virttest/utils_net.py

```diff
@@ -4322,3 +4323,30 @@ def delete_ovs_bridge(ovs_bridge_name, session=None, ignore_status=False):
     return utils_misc.cmd_status_output(tmux_cmd, shell=True, verbose=True,
                                         ignore_status=ignore_status,
                                         session=session)
+
+
+def get_iface_info(mac, session=None):
```

Comment:
Considering that this is still similar but a more detailed version of `get_linux_ifname`, could we place it right after `get_linux_ifname` instead of at the end of the module? As this module contains windows methods too we should also give it more comparable naming like `get_linux_iface_info`.